File: src/main/java/org/shark/example/service/quartz/pojo/CreateJobDto.java
package org.shark.example.service.quartz.pojo;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

@Data
public class CreateJobDto {
    private String jobName;
    private String jobGroup;
    @Schema(example = "0 0/5 * * * ?")
    private String cronExpression;
}

File: src/main/kotlin/org/shark/example/service/quartz/pojo/CreateJobDto.kt
package org.shark.example.service.quartz.pojo

import io.swagger.v3.oas.annotations.media.Schema

data class CreateJobDto(
    var jobName: String? = null,
    var jobGroup: String? = null,
    @Schema(example = "0 0/5 * * * ?")
    var cronExpression: String? = null
)
